### DISCUSSION TITLE
iCloud Sync on iOS devices

### DISCUSSION THREAD

**Discussion by zfl0w** [osmandapp/OsmAnd#12700] [Ideas]
For many different usecases, I can imagine users would like to have synchronized data and configurations on different devices. Like, for example, plan a hiking tour kn iPad and view it on iPhone. Or, mark some POI on the go and view it later on another.

What do you think about synchronizing the whole app configuration using iCloud?

**Comment by vshcherb** [comment]
Well we need to check if this doesn't work already, anyway we plan to work on it Q3-Q4 2021.
